spec/javascript/reviewsContainer.js

```diff
@@ -0,0 +1,18 @@
+import testHelper from "./testHelper";
+import ReviewsContainer from "../../app/javascript/react/components/ReviewsContainer";
+
+describe("ReviewsContainer", () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = mount(
+      <ReviewsContainer
+        memeId={3}
+      />
+    );
+  });
+
+  it('should render a div element with the class of \'reviews-container\'', () => {
```

Comment:
Please also check that you have at least one `ReviewTile` displaying -- your `showContainer.js` test does a good job of modeling this. Note that you'll need to fetchMock a get to replace line 14 of ReviewsContainer in order to give yourself some reviews from which to make tiles 🙂